api/net/Dockerfile

```diff
@@ -0,0 +1,35 @@
+ARG BUILD_CONFIGURATION=Release
+FROM mcr.microsoft.com/dotnet/aspnet:6.0 AS base
+EXPOSE 443 80
+
+# Copy csproj and restore as distinct layers
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS build
+WORKDIR /src/api/net
+COPY api/net/ .
+COPY libs/net/ /src/libs/net/
+
+RUN dotnet restore
+ENV PATH="$PATH:/root/.dotnet/tools"
+RUN dotnet tool install --global dotnet-ef --version 6.0.3
+
+RUN dotnet build "TNO.API.csproj" -c "$BUILD_CONFIGURATION" -o /app/build
+
+FROM build AS publish
+RUN dotnet publish "TNO.API.csproj" -c "$BUILD_CONFIGURATION" -o /app/publish
+
+# Runtime image
+FROM base AS final
+
+RUN apt-get update && apt-get -y upgrade
+RUN apt install curl -y
+
+WORKDIR /app
+COPY --from=publish /app/publish .
+COPY api/net/entrypoint.sh .
+RUN chmod +x /app/entrypoint.sh
+
+RUN useradd --create-home appuser
+USER appuser
```

Comment:
Not sure if this will run in Openshift yet, but I'll test once we have this merged.